Update introduction.adoc

thomas-saigre opened the pull request:
Add json specifications for boundaries conditions
also minor typo fixes

@prudhomm @vincentchabannes can you please review that this additions are corrects ? I'm not sure in particular of `expr1` and `expr2` for Robin conditions

- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/feelpp/toolbox/pulls) for the same change?
- [ ] Have you fixed a Bug ?
- [ ] Have you added the documentation of the example or benchmark ?
- [ ] Have you successfully run the example or benchmark?
- [ ] Have you documented the results?
- [ ] Have you added the images to the documentation?
- [ ] Have you added the vtkjs files for 3D visualization in the documentation?
- [ ] Have you added the vtkjs files to a Data Management Plateform _e.g_ https://girder.math.unistra.fr

-----

thomas-saigre committed:
Update introduction.adoc

Add json specifications for boundaries conditions
also minor typo fixes

@prudhomm @vincentchabannes can you please review that this additions are corrects ? I'm not sure in particular of `expr1` and `expr2` for Robin conditions

prudhomm commented on toolboxes/modules/cfpdes/partials/introduction.adoc:
```diff
@@ -233,7 +233,29 @@ u_i = g_i(x,t,\mu), \quad i=1,\dots,N
+----
+****
+
+<1> Keyword for Dirichlet conditions.
```
replace with `<.>` it will follow the order above

thomas-saigre replied:
done in c6df538df2ccf52fdc470f6dcb74c3c178b767d8

prudhomm commented on toolboxes/modules/cfpdes/partials/introduction.adoc:
```diff
@@ -233,7 +233,29 @@ u_i = g_i(x,t,\mu), \quad i=1,\dots,N
+****
+
+<1> Keyword for Dirichlet conditions.
+<2> The physical entity (associated to the mesh) to which the condition is applied.
+<3> The markers of the boundary where the Dirichlet condition is applied.
+<4> The expression for the Dirichlet stem:[g_i]. See more details in xref:toolboxes:ROOT:modeling-analysis-using-json-files.adoc#_boundaryconditions[the json specifications].
```
see above

prudhomm commented on toolboxes/modules/cfpdes/partials/introduction.adoc:
```diff
@@ -264,6 +286,25 @@ The shape of the Neumann condition is the same as the unknown shape.
+----
+****
+
+<1> Keyword for Robin conditions.
+<2> The physical entity (associated to the mesh) to which the condition is applied.
+<3> Expression of stem:[g_i]. See more details in xref:toolboxes:ROOT:modeling-analysis-using-json-files.adoc#_boundaryconditions[the json specifications].
+
```
replace with `<.>`

prudhomm commented on toolboxes/modules/cfpdes/partials/introduction.adoc:
```diff
@@ -292,6 +333,27 @@ The shape of the Robin condition is the same as the unknown shape.
+----
+****
+
+<1> Keyword for Robin conditions.
+<2> The physical entity (associated to the mesh) to which the condition is applied.
+<3> Expression of stem:[\eta_i]. See more details in xref:toolboxes:ROOT:modeling-analysis-using-json-files.adoc#_boundaryconditions[the json specifications].
+<4> Expression of stem:[\zeta_i].
```
replace with `<.>`

prudhomm commented:
@thomas-saigre I propose some minor changes

thomas-saigre committed:
replace item with <.>
/cc @prudhomm

prudhomm committed:
Merge branch 'develop' into thomas-saigre-patch-1

prudhomm commented:
@thomas-saigre @vincentchabannes the PR deploy using netlify should appear any time soon

prudhomm commented:
![screenshot](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiNDAyNmY4M2QtNTNhMS00YWQ2LTg2ZTctYjZiZGE0Mjk4Y2M2IiwiZGVwbG95X2lkIjoiNjRkZGRkY2QzMTQxZWUwMDA4YjYwMDIzIiwiaWQiOiI4YTMxYzlkNy1lY2ZhLTQ2NzEtOWZjMi1kNTI2MzVkMTVkY2UifQ==)

@thomas-saigre please could you `code`add some explanation for initial conditions as well ?


<details><summary>Browser metadata</summary>





```
Path:      /toolboxes/latest/cfpdes/manual/
Browser:   Chrome 115.0.0.0 on Mac OS 10.15.7
Viewport:  1728 x 961 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-180--dashing-twilight-9661e1.netlify.app%2Ftoolboxes%2Flatest%2Fcfpdes%2Fmanual%2F&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=115.0&resolution=1728x961&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-180--dashing-twilight-9661e1.netlify.app/toolboxes/latest/cfpdes/manual/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=64ddddcd3141ee0008b60023&commentID=64dde3290c8ef05fb19128ea&resolution=resolved)

prudhomm commented:
![screenshot](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiNDAyNmY4M2QtNTNhMS00YWQ2LTg2ZTctYjZiZGE0Mjk4Y2M2IiwiZGVwbG95X2lkIjoiNjRkZGRkY2QzMTQxZWUwMDA4YjYwMDIzIiwiaWQiOiIyM2U1ZjhmNS02MWNhLTQzOGEtOGEyOC02ZmU0MTZmMjQ1NTQifQ==)

perhaps a good idea to have different kind of conditions for different border to examplify ?

<details><summary>Browser metadata</summary>





```
Path:      /toolboxes/latest/cfpdes/manual/
Browser:   Chrome 115.0.0.0 on Mac OS 10.15.7
Viewport:  1728 x 961 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-180--dashing-twilight-9661e1.netlify.app%2Ftoolboxes%2Flatest%2Fcfpdes%2Fmanual%2F&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=115.0&resolution=1728x961&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-180--dashing-twilight-9661e1.netlify.app/toolboxes/latest/cfpdes/manual/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=64ddddcd3141ee0008b60023&commentID=64dde3aa822d0c5a172e9b3f&resolution=resolved)

prudhomm commented:
![screenshot](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiNDAyNmY4M2QtNTNhMS00YWQ2LTg2ZTctYjZiZGE0Mjk4Y2M2IiwiZGVwbG95X2lkIjoiNjRkZGRkY2QzMTQxZWUwMDA4YjYwMDIzIiwiaWQiOiI2NzkzZjg1My04NzJlLTQ4ZDMtODBlYy01ZjNmZWEyZGMwOGEifQ==)

perhaps add a comment that this will be considered a non-linear condition due to the presence of `heat_T``code` ? an alternative to avoid that is to use `heat_T_rt` (@vincent is this correct?) so that `heat_T` from previous  iteration is used(time or non-linear)

<details><summary>Browser metadata</summary>





```
Path:      /toolboxes/latest/cfpdes/manual/
Browser:   Chrome 115.0.0.0 on Mac OS 10.15.7
Viewport:  1728 x 961 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-180--dashing-twilight-9661e1.netlify.app%2Ftoolboxes%2Flatest%2Fcfpdes%2Fmanual%2F&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=115.0&resolution=1728x961&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-180--dashing-twilight-9661e1.netlify.app/toolboxes/latest/cfpdes/manual/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=64ddddcd3141ee0008b60023&commentID=64dde45c26417b5f27843fea&resolution=resolved)

prudhomm commented:
![screenshot](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiNDAyNmY4M2QtNTNhMS00YWQ2LTg2ZTctYjZiZGE0Mjk4Y2M2IiwiZGVwbG95X2lkIjoiNjRkZGRkY2QzMTQxZWUwMDA4YjYwMDIzIiwiaWQiOiJhNGYwMzFhYS1kMDIwLTRlMjAtYWRjYy03MmJhMWZkNDMwN2IifQ==)

stress out that Robin provide a linear boundary condition so term implying non-linearity cannot be used. Again same trick with `_rt` (remove trial) could be used to avoid that  

<details><summary>Browser metadata</summary>





```
Path:      /toolboxes/latest/cfpdes/manual/
Browser:   Chrome 115.0.0.0 on Mac OS 10.15.7
Viewport:  1728 x 961 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-180--dashing-twilight-9661e1.netlify.app%2Ftoolboxes%2Flatest%2Fcfpdes%2Fmanual%2F&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=115.0&resolution=1728x961&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-180--dashing-twilight-9661e1.netlify.app/toolboxes/latest/cfpdes/manual/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=64ddddcd3141ee0008b60023&commentID=64dde4e6f3a3b46078cb81d6&resolution=resolved)

thomas-saigre committed:
remove trailing spaces

thomas-saigre committed:
add more details in code #180
/cc @prudhomm @vincentchabannes
can you please review that what I added is correct ?

thomas-saigre committed:
Merge remote-tracking branch 'origin/develop' into thomas-saigre-patch-1

thomas-saigre committed:
fix tables errors

thomas-saigre committed:
other small fixes
[ci skip]

vincentchabannes commented on toolboxes/modules/cfpdes/partials/introduction.adoc:
```diff
@@ -104,17 +104,22 @@ NOTE: In the following, `mu` is a parameter defined e.g. in the `Parameters` sec
-                    "basis":"Pch1", // the unknown is a scalar field
-                    "name":"u",
-                    "symbol":"u"
+        "myscalarpde": <2>
```
I prefer the compact version. More standard for JSON
